python_ta/cfg/graph.py

```diff
@@ -190,17 +201,25 @@ class CFGEdge:
     """An edge in a control flow graph.
 
     Edges are directed, and in the future may be augmented with auxiliary metadata about the control flow.
+
+    Condition stores the AST node representing the condition tested in If and While statements.
```

Comment:
The capitalization here is a bit confusing. You could do simply "condition stores...", or in more natural English "The condition attribute stores..."